ansible/roles/dev-desktop/files/scripts/link_rust.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Enable strict mode for Bash
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+IFS=$'\n\t'
```

Comment:
Personally, I prefer repetition here since it makes the flags very explicitly. We can replace it with something like `source ./enable-strict-mode.sh`. But I'm afraid that no-one will know what that actually does, leading to confusion and frustration when debugging or extending a script.